src/platforms/windows/windowscommons.cpp

```diff
@@ -67,22 +70,43 @@ QString WindowsCommons::tunnelConfigFile() {
   return QString();
 }
 
+// static
 QString WindowsCommons::tunnelLogFile() {
-  QStringList paths =
-      QStandardPaths::standardLocations(QStandardPaths::GenericDataLocation);
+  static QString tunnelLogFilePath = getTunnelLogFilePath();
```

Comment:
for my understanding, are you declaring `tunnelLogFilePath` as static because we don't need to recalculate the path beyond the first initialization?

Reply:
Yes, this is an optimization to reduce file system activity because log file path doesn't change.